### PR TITLE
Retrospective Consistency on RDM chapter of Guide to Reproducible Research

### PR DESCRIPTION
**Pull request by paulowoicho**
### Summary

Related to #1174, moved from #1396

This PR addresses some of the consistency issues highlighted in issue 1174 for the Research Data Management chapter of the Guide to Reproducible Research. (This branch is up to date with my master branch, so it carries those changes as well)

### List of changes proposed in this PR (pull-request)

- [ ] * Ensuring the names of chapters/subchapters map exactly to the left TOC.
- [ ] * Ensuring consistency with the Turing Way Style Guide
- [ ] * Ensuring correct grammar and a consistent tone (with the help of Grammarly)
- [ ] * Making chapters, subchapters, and sections of the book 'reference-able' by adding labels.
- [ ] * Converting all HTML formatting to markdown, where possible
- [ ] * Ensuring proper title casing for headers
- [ ] * Ensuring chapters follow the structure discussed in #1174 

### What should a reviewer concentrate their feedback on?

- [ ] Do the changes make sense and does everything look ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
I have tagged @EstherPlomp and @Ismael-KG to review this, whoever has some time and can check your contribution.

File: book/website/reproducible-research/rdm/rdm-spreadsheets.md [hunk 1]
```diff
@@ -140,45 +158,48 @@ iris %>% ## the iris dataset is included in R base
...
-When aiming at developing a reproducible workflow for analyses, one should design the spreadsheet for both computer and human readability, and even before starting data collection, they should consider what makes their data analysis easy. 
+While spreadsheets can be a very user-friendly way to collect and share data, they can also be the source of mistakes if misused. 
+When aiming at developing a reproducible workflow for analyses, one should design the spreadsheet for both computer and human readability, and, even before starting data collection, they should consider what would make their data analysis easy.
 Data manipulation and analysis in spreadsheets, in particular, is best avoided as it leads to non-reproducible workflows. 
 Using version control and making the data read-only are two additional data management practices that can prevent accidents.
 
-Use a README file to explain naming conventions and other design choices on top of the spreadsheet.
-It will make it clear to others what the file names mean and what criteria to consider when designing an analysis workflow. 
+Use a README file and other design choices to explain naming conventions.
```
Comment (EstherPlomp) [inline book/website/reproducible-research/rdm/rdm-spreadsheets.md:201]: to explain naming convention _and other structure choices._

File: book/website/reproducible-research/rdm/rdm-spreadsheets.md [hunk 2]
```diff
@@ -140,45 +158,48 @@ iris %>% ## the iris dataset is included in R base
...
+It will make it clear to others what the header names mean and what criteria to consider when designing an analysis workflow.
```
Comment (EstherPlomp) [inline book/website/reproducible-research/rdm/rdm-spreadsheets.md:202]: what the _file and_ header names mean

**Review by EstherPlomp — CHANGES_REQUESTED** [review]
Hi @paulowoicho. Thanks so much for all of that! I only have some minor suggestions for change before merging.

File: book/website/reproducible-research/rdm/rdm-stories.md [hunk 3]
```diff
@@ -17,34 +17,30 @@ Let's start with my digital footprints so that you can discovery my activities:
 I have worked on research data management since 2001, yes, way before this area was even considered a 'thing'.
 I was also told that there is no career to make in research data management.
 Well, how wrong that comment was!
-Formerly seen as intersection between service provision and IT, data management has become a first class citizen, recognised as a research and development subject, as it should be.
+Formerly seen as an intersection between service provision and IT, data management has become a first-class citizen, recognized as a research and development subject, as it should be.
```
[the remark this answers is not in the document]
Comment (paulowoicho) [reply book/website/reproducible-research/rdm/rdm-stories.md:20]: It's British spelling! I must have changed my Grammarly settings at this point

Comment (paulowoicho) [inline book/website/reproducible-research/rdm/rdm-stories.md:20]:
```suggestion
Formerly seen as an intersection between service provision and IT, data management has become a first-class citizen, recognised as a research and development subject, as it should be.
```

**Comment by paulowoicho** [comment]
Thank you @EstherPlomp for the review! I have made the edits you suggested

**Review by EstherPlomp — APPROVED** [review]
Great, no further comments from my side! Cheers! :)

**Pull request merged**